npm-debug.log

```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```

Comment:
You commented also unnecessary `npm-debug.log`. I added it to `.gitignore` a4342a3, but you should check pull request diff next time.